src/components/GuideTile/GuideTile.js

```diff
@@ -16,33 +15,133 @@ const GuideTile = ({
   className,
   children,
   alignment,
+  featured,
   ...props
 }) => (
-  <Component
+  <Surface
     {...props}
-    className={cx(styles.tile, className, {
-      [styles.tileWithIcon]: icon,
-      [styles.tileLeftAligned]: alignment === GuideTile.ALIGNMENT.LEFT,
-      [styles.tileCenterAligned]: alignment === GuideTile.ALIGNMENT.CENTER,
-    })}
+    as={Component}
+    className={className}
+    base={Surface.BASE.PRIMARY}
+    css={css`
+      display: grid;
+      grid-template-rows: auto auto 1fr auto;
+      border-radius: 0.25rem;
+      position: relative;
+      padding: 1rem;
+      transition: all 0.15s ease-out;
+
+      &:hover {
+        transform: translateY(-2px);
+        border-color: var(--border-hover-color);
+      }
+
+      ${icon &&
+      `
+        h2 {
+          margin-top: 1rem;
+        }
+      `}
+
+      ${featured &&
+      `
+          border-color: var(--color-white);
+
+          .dark-mode & {
```

Comment:
We should try and leave the border/background styles to the `Surface`, since its goal is to handle the box shadow/border/background on surfaces rendered "above" other content. 

In this case, it looks like you're setting these background/border colors based on whether these are rendered on the featured section. Rather than conditionally applying styles, apply a different `base` to the surface. In this case, we want to render the surface on a "secondary" base when it's a featured tile. Remove these styles in favor of: `base={featured ? Surface.BASE.SECONDARY : Surface.BASE.PRIMARY}`